src/main/webapp/app/appConfig.ts

```diff
@@ -1,5 +1,6 @@
 type FrontendProperties = {
   firebase: FirebaseProperties;
+  sentryDsn: string;
```

Comment:
Could you help make the FrontendProperties optional and also `firebase` and `sentryDsn` optional in the type. We may have a scenario where we don't enable Firebase or Sentry or both. In that case, these properties can be undefined.